<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="view_tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Learning rate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="learning_rate">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="minimum">
             <double>0.000100000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>momentum</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="momentum">
            <property name="maximum">
             <double>0.990000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.900000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>batch size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="batch_size">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>1024</number>
            </property>
            <property name="value">
             <number>64</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>epoch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="epoch">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>5000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rotate_sample">
            <property name="text">
             <string>rotation samples</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Train</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reset_nn">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="nn">
          <property name="text">
           <string>32,32,1|conv5,relu|32,32,6|max_pooling2|16,16,6|conv3,relu|16,16,16|max_pooling2|8,8,16|full,relu|1,1,120|full,relu|1,1,84|full|1,1,10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSlider" name="data_pos">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="data_view"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="classification_error">
            <property name="text">
             <string>error calculation</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="error_table"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTextBrowser" name="network_list"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="save_list">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="generate_list">
            <property name="text">
             <string>gen list</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_data"/>
    <addaction name="actionOpen_testing_data"/>
    <addaction name="actionOpen_network"/>
    <addaction name="separator"/>
    <addaction name="actionSave_training_data"/>
    <addaction name="actionSave_testing_data"/>
    <addaction name="actionSave_network"/>
   </widget>
   <widget class="QMenu" name="menu_Data">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <addaction name="actionCopy_X_Flip"/>
    <addaction name="actionCopy_Y_Flip"/>
    <addaction name="actionCopy_XY_Swap"/>
    <addaction name="actionAdd_data"/>
    <addaction name="actionIntensity_normalization"/>
    <addaction name="action10_fold_Data_Deparation"/>
    <addaction name="actionPadding_rare_labels"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Data"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_data">
   <property name="text">
    <string>Open training data...</string>
   </property>
  </action>
  <action name="actionOpen_network">
   <property name="text">
    <string>Open network...</string>
   </property>
  </action>
  <action name="actionOpen_testing_data">
   <property name="text">
    <string>Open testing data...</string>
   </property>
  </action>
  <action name="actionSave_training_data">
   <property name="text">
    <string>Save training data...</string>
   </property>
  </action>
  <action name="actionSave_testing_data">
   <property name="text">
    <string>Save testing data...</string>
   </property>
  </action>
  <action name="actionCopy_X_Flip">
   <property name="text">
    <string>Copy X Flip</string>
   </property>
  </action>
  <action name="actionCopy_Y_Flip">
   <property name="text">
    <string>Copy Y Flip</string>
   </property>
  </action>
  <action name="actionCopy_XY_Swap">
   <property name="text">
    <string>Copy XY Swap</string>
   </property>
  </action>
  <action name="actionSave_network">
   <property name="text">
    <string>Save network...</string>
   </property>
  </action>
  <action name="actionAdd_data">
   <property name="text">
    <string>Add Data...</string>
   </property>
  </action>
  <action name="actionIntensity_normalization">
   <property name="text">
    <string>Intensity normalization</string>
   </property>
  </action>
  <action name="action10_fold_Data_Deparation">
   <property name="text">
    <string>10-fold Data Deparation</string>
   </property>
  </action>
  <action name="actionPadding_rare_labels">
   <property name="text">
    <string>Padding rare labels</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
